registry/server/app.ts

```diff
@@ -25,7 +25,7 @@ export default async (withAuth: boolean = true) => {
 
     app.get('/ping', pong);
 
-    app.use('/', serveStatic('client/dist'));
+    app.use('/', serveStatic('client/dist') as any);
```

Comment:
Is "as any" used as a temporary solution?

Reply:
nope, types are not compatible unfortunately